### PR TITLE
added logo to the header

### PR DESCRIPTION
**Pull request by sun-amy**
Moved the logo from the public folder into src/ImageAssets folder to be accessible, then added the logo to the header.

image

We also moved the header.tsx file into its own folder, and the Hamburgerbutton.tsx and the Hamburgerbutton.scss into their own folder.

Additionally we changed the tab title to Marsio Kart.

image

## Things to Check
- [x] Have all the requirements of your ticket have been met
- [ ] Have you added tests if needed for your ticket - NA
- [ ] Are all the tests passing - NA

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Marsio Kart</title>
```
Comment (Kiran-R-K) [inline public/index.html:27]: Nice change :)

File: src/QuestionDisplay/QuestionDisplay.scss
Comment (Kiran-R-K) [inline src/QuestionDisplay/QuestionDisplay.scss:1]: I think this file was accidentally left in your branch form another ticket - you can delete it from this branch so it doesn't get merged in with this work

File: src/QuestionDisplay/QuestionDisplay.tsx
Comment (Kiran-R-K) [inline src/QuestionDisplay/QuestionDisplay.tsx:1]: As above, this looks like this file has snuck in from another branch - you can delete it from this branch so it doesn't get merged in with this work

File: src/Header/Header.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ const Header: React.FC = () => {
   return (
     <div className="App">
       <Hamburger onClick={handleHamburgerClick} label={""}/>
+      <img src={require('../ImageAssets/marsiokartlogo.jpg')} />
```
Comment (Kiran-R-K) [inline src/Header/Header.tsx]: How would you feel about saving this as a constant(e.g. `const logo = require('../ImageAssets/marsiokartlogo.jpg')`) and then adding it into the image tag by it's variable name?
I think it's worth considering for readability but I'm also happy with whichever choice you make here.

**Review by Kiran-R-K — APPROVED** [review]
This looks really good - I also appreciate that you've taken the time to do some clean up and improvement of file structuring around the app.

A couple of points:
- I have left an optional comment, I intend to approve either way but it's up to you two if you want to action it or not - see how you feel about it.
- I'm going to approve this but it's not mergeable yet because the pipeline has failed so once you push a fix for that you will be able to merge it without it needing another review. (I'm happy to talk through what the error is and the fix for it if you want just pop me a message).
- Moving those files into folders could to lead to some clashes with other code (that doesn't mean it's not a good choice -  it is, but it's one that requires extra team communication) I would suggest sending a message in the mars-mission channel letting everyone know what you moved and where so that they know to expect this change when they merge into main.

Let me know if you have any questions

File: src/Header/Header.tsx (hunk 2 above)
Comment (sun-amy) [reply src/Header/Header.tsx]: I have implemented this.

**Pull request merged**